ui/src/app/applications/components/application-node-info/application-node-info.scss

```diff
@@ -39,4 +39,28 @@
             color: $argo-color-gray-8;
         }
     }
+
+    &__container {
+        display: flex;
+        align-items: center;
+        flex-direction: row;
+        line-height: 1.8;
+        border-bottom: 1px solid rgba(222, 230, 235, 0.7);
+
+        &--name {
+            width: 15%;
+        }
+        &--highlight {
+            font-style: italic;
+        }
+
+        &--hint {
+            text-decoration: underline;
+            text-decoration-style: dashed;
+            cursor: default;
```

Comment:
```suggestion
            cursor: pointer;
            
            &:hover {
                text-decoration: none;
            }
```

The tooltip is kinda slow to load. I think these two changes would help the user know that they're waiting for the tooltip to appear.